internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestActiveActiveIngestion.java

```diff
@@ -127,6 +127,7 @@ public void setUp() {
     Properties serverProperties = new Properties();
     serverProperties.setProperty(ConfigKeys.SERVER_PROMOTION_TO_LEADER_REPLICA_DELAY_SECONDS, Long.toString(1));
     serverProperties.put(ROCKSDB_PLAIN_TABLE_FORMAT_ENABLED, false);
+    serverProperties.put(SERVER_INGESTION_TASK_THREAD_SAFE_MODE, false);
```

Comment:
Since this mode is default false, can we make it to run both true and false for some sophisticated tests for AAWC?